### PR TITLE
Query multiple tables

### PR DESCRIPTION
**Pull request by hopeforkoami**
In this Branch, I:
For the third milestone of the Vet Clinic database project, I:
 - Created a table named owners with columns id, full_name, and age
 - Created a table named species with columns id and name
 - Modified the animals table
 - Removed column species
 - Added column species_id which is a foreign key referencing the species table
 - Added column owner_id which is a foreign key referencing the owners table
 - Inserted the given data into the owners table
 - Inserted the given data into the species table
 - Altered the animals table with the given data
 - Modified the inserted animals to include values of species_id and owners_id
 - Performed queries using JOIN

**Screenshots of the schema update**

update_schema

update_schema_child1

**Screenshots of Data update**

add_data

update_data_animalis_owners

update_data_animalis_species

**Screenshots of the queries**

queries_part1

queries_part2

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -19,4 +19,22 @@ insert into animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
 insert into animals (name, date_of_birth, escape_attempts, neutered, weight_kg) values
 ('Blossom', '1998-10-13', 3, true, 17);
 insert into animals (name, date_of_birth, escape_attempts, neutered, weight_kg) values
-('Ditto', '2022-05-14', 4, false, 22);
\ No newline at end of file
+('Ditto', '2022-05-14', 4, true, 22);
+
+insert into owners (full_name, age) values ('Sam Smith',34),
+('Jennifer Orwell',19),
+('Bob', 77),
```
Comment (fobadara) [inline data.sql]: [REQUIRED] Bob's age should be 45

**Review by fobadara — CHANGES_REQUESTED** [review]
Hi @hopeforkoami , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights
✅ Linters are passing

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (hopeforkoami) [reply data.sql]: Thanks

**Review by cynthiainga — APPROVED** [review]
## STATUS APPROVED ✔️ ✔️ ✔️ 

Hi @hopeforkoami, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- [ ] Descriptive PR ✔️ 
- [ ] Correct Gitflow ✔️ 
- [ ] Good commit history ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**